src/chat/ChannelsList.js

```diff
@@ -93,6 +94,12 @@ const ChannelsList = themed((props) => {
     watch: true,
   };
 
+  const sinceYesterday = moment().subtract(24, 'hour');
```

Comment:
I know I originally suggested the 24hr time, but what if we shrink it to 12 instead?

Thinking if there are livestream services on Saturday and then again Sunday morning, there's a chance 24hrs isn't long enough for the "old" chats to hide.

WDYT?

```suggestion
  const sinceYesterday = moment().subtract(12, 'hour');
```